.github/workflows/prebuilt_binaries.yml

```
@@ -323,13 +323,38 @@ jobs:
             exit 1
           fi
 
-      - name: MPI validation
+      - name: Additional validation (MPI and uninstall)
         uses: ./.github/actions/run-in-docker
         with:
           image: validation:local
           user: cudaq
           shell: bash -l
           run: |
+            # Uninstall CUDA Quantum
+            sudo bash ${CUDA_QUANTUM_PATH}/uninstall.sh -y
+            llvm_dir=`bash -c 'source build_environment.sh && echo $LLVM_INSTALL_PREFIX'`
+            cuquantum_dir=`bash -c 'source build_environment.sh && echo $CUQUANTUM_INSTALL_PREFIX'`
+            cutensor_dir=`bash -c 'source build_environment.sh && echo $CUTENSOR_INSTALL_PREFIX'`
+            if [ -d "$llvm_dir" ] || [ -d "$llvm_dir" ] || [ -d "$llvm_dir" ] ||  [ -d "$CUDA_QUANTUM_PATH" ]; then
```

This looks like duplicate checks for `$llvm_dir`. Is it supposed to be this instead?
```suggestion
            if [ -d "$llvm_dir" ] || [ -d "$cuquantum_dir" ] || [ -d "$cutensor_dir" ] ||  [ -d "$CUDA_QUANTUM_PATH" ]; then
```